query-service/src/integrationTest/java/org/hypertrace/core/query/service/htqueries/HTPinotQueriesTest.java

```diff
@@ -95,7 +95,7 @@ public static void setup() throws Exception {
     kafkaZk.start();
 
     pinotServiceManager =
-        new GenericContainer<>(DockerImageName.parse("hypertrace/pinot-servicemanager:main"))
+        new GenericContainer<>(DockerImageName.parse("hypertrace/pinot-servicemanager:0.6.12"))
```

Comment:
Filed a tracking ticket to make it work it with main version. Once fixed, will revert this change